src/hotspot/share/jfr/support/jfrThreadLocal.cpp

```diff
@@ -78,7 +80,15 @@ JfrThreadLocal::JfrThreadLocal() :
   _enqueued_requests(false),
   _vthread(false),
   _notified(false),
-  _dead(false) {
+  _dead(false)
+#ifdef LINUX
+  ,_cpu_timer(nullptr),
+  _cpu_time_jfr_locked(UNLOCKED),
+  _has_cpu_time_jfr_requests(false),
+  _cpu_time_jfr_queue(0),
+  _wants_is_thread_in_native_stackwalking(false)
```

Comment:
This field name can be improved.

"_do_async_processing_of_cpu_time_jfr_requests"?